Movies Time/ViewController/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,250 @@
+//
+//  MovieDetailsViewController.swift
+//  Movies Time
+//
+//  Created by Jose Eduardo on 07/09/21.
+//
+
+import UIKit
+
+class MoviesDetailsViewController: UIViewController {
+    
+    enum Constants {
+        static let padding: CGFloat = 10
+    }
+    
+    var movie: Movie?
+    var id: Int?
+    
+    var movieBackgroundView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
if you are going to create different views with the same properties, it is better to create a method so you don't duplicate too much code